dwave/cloud/auth/flows.py

```diff
@@ -205,6 +212,39 @@ def refresh_token(self):
         self._save_token_to_creds(token)
         return token
 
+    def revoke_token(self, *,
+                     token: Optional[str] = None,
+                     token_type_hint: Optional[
+                         Literal['access_token', 'refresh_token']] = None
+                     ) -> bool:
+        """Revoke OAuth 2.0 token.
+
+        Args:
+            token:
+                OAuth 2.0 access or refresh token to be revoked. If refresh token
+                is specified, the authorization server should also revoke the
+                corresponding access tokens. If ``token`` is not specified, then
```

Comment:
From "If ``token`` is not specified, then current refresh token is revoked" I'd understand that `revoke_token(token_type_hint='access_token')` will revoke the refresh token: is that correct?

Reply:
No, that's just a _hint_ for authorization server. If user doesn't specify `token`, the client library tries using refresh token, and if refresh token is not available (should never really happen here), then access token is used.